src/middleware/csrfErrorMiddleware.ts

```diff
@@ -0,0 +1,24 @@
+import { ErrorRequestHandler, NextFunction, Request, Response } from 'express'
+import {
+    CsrfError
+} from '@companieshouse/web-security-node'
+
+// TODO: Enter the template name here instead of <TEMPLATE NAME>
+const csrfErrorTemplateName = "403";
```

Comment:
Consider a more meaningful template name. These are just for devs so how about `csrf-error`

Reply:
thanks @SteveHicksCH I have modified to be consistent with existing middleware files.